src/components/SkipLink.tsx

```diff
@@ -1,43 +1,31 @@
 import React from "react"
-import styled from "@emotion/styled"
 import Translation from "./Translation"
-
-const Div = styled.div`
-  background-color: ${(props) => props.theme.colors.primary};
-`
-
-const Anchor = styled.a`
-  line-height: 2rem;
-  position: absolute;
-  top: -3rem;
-  margin-left: 0.5rem;
-  color: ${(props) => props.theme.colors.background};
-  text-decoration: none;
-
-  &:focus {
-    position: static;
-  }
-`
-
-const DivAnchor = styled.div`
-  height: 80px;
-  margin-top: -80px;
-`
+import { Box, Link } from "@chakra-ui/react"
 
 export interface IProps {
   hrefId: string
 }
 
 export const SkipLink: React.FC<IProps> = ({ hrefId }) => {
   return (
-    <Div>
-      <Anchor href={hrefId}>
+    <Box bg="primary">
+      <Link
+        href={hrefId}
+        lineHeight="2rem"
```

Comment:
https://chakra-ui.com/docs/styled-system/theme#typography
```suggestion
        lineHeight="taller"
```